doc/configuration.rst

```diff
@@ -890,6 +890,10 @@ to use the 2nd figure created as the thumbnail::
 
     # sphinx_gallery_thumbnail_number = 2
 
+To use the last figure created as the thumbnail::
```

Comment:
Maybe make it more clear that negative counts from the last. e.g.:

"You can also use negative numbers, which counts from the last figure. For example `-1` means use the last figure created in the example as the thumbnail."

Reply:
Done in 2734232dcd746f18d7b2818678b9d905113f2e3e.